exporter/exporter.py

```diff
@@ -22,9 +21,22 @@ def export_metrics(port=8000):
 
     while True:
 
-        # cpu_utilization_gauge.set(cpu_collector.get_utilization())
+        set_gauge(cpu_utilization_gauge, cpu_collector.get_utilization())
+        set_gauge(cpu_frequency_gauge, cpu_collector.get_frequency())
+        set_gauge(cpu_temperature_gauge, cpu_collector.get_temperature())
 
-        ram_utilization_gauge.set(ram_collector.get_utilization())
-        ram_memory_gauge.set(ram_collector.get_memory())
+        set_gauge(ram_utilization_gauge, ram_collector.get_utilization())
+        set_gauge(ram_memory_gauge, ram_collector.get_memory())
+        set_gauge(cpu_temperature_gauge, cpu_collector.get_temperature())
 
-        time.sleep(5)
\ No newline at end of file
+        time.sleep(5)
+
+
+def set_gauge(gauge=Gauge, value=float):
+
+    """
+    Sets the gauge to the specified value if the value is not None.
+    """
+    
+    if value is not None:
+        gauge.set(value)
```

Comment:
@mccooeyc11 
I refactored `exporter.py` to check for None values before setting Gauges. This is a sensible thing to do to avoid null pointer errors.